### PR TITLE
doc: a small facelift

### PR DESCRIPTION
**Pull request by bzz**
Small updates:
 - update all image names
 - slides link TBD
 - add notebook links that work on GH as well
 - update workshop description to match the published one

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,31 +9,25 @@ OSS tools covered:
 - [gitbase](https://docs.sourced.tech/gitbase)
 - [bblfsh](https://doc.bblf.sh)
 - [BigARTM](http://bigartm.org)
-- [OpenNMT](http://opennmt.net)
+- [PyTourch](https://pytorch.org)
```
Comment (m09) [inline README.md]:
```suggestion
- [PyTorch](https://pytorch.org)
```

File: README.md [hunk 2]
```diff
@@ -9,31 +9,25 @@ OSS tools covered:
...
 
  <details>
 <summary>Abstract</summary>
 
-> Machine Learning on Source Code (MLonCode) is an emerging and exciting research domain which stands at the sweet spot between deep learning, natural language processing, social science, and programming.
+> Machine Learning on Source Code (MLonCode) is an emerging research domain which stands at the > intersection of deep learning, natural language processing, software engineering and programming > language communities.
```
Comment (m09) [inline README.md]:
```suggestion
> Machine Learning on Source Code (MLonCode) is an emerging research domain which stands at the intersection of deep learning, natural language processing, software engineering and programming language communities.
```

File: README.md [hunk 3]
```diff
@@ -9,31 +9,25 @@ OSS tools covered:
...
 >
-> During this 2 hours workshop, we are going to show you how to extract insights from code bases—step by step—by shedding light on those crucial aspects:
+> During this 3.5 hours workshop, we will review the recent SE tasks that benefit from applying ML and focus the hands-on experience on:
```
Comment (m09) [inline README.md]:
```suggestion
> During this 3h30 workshop, we will review recent Software Engineering tasks that benefit from applying Machine Learning, with a focus on hands-on experience on:
```

File: README.md [hunk 4]
```diff
@@ -9,31 +9,25 @@ OSS tools covered:
...
+> - extracting data from the real source code and
```
Comment (m09) [inline README.md]:
```suggestion
> - extracting data from real source code
```

File: README.md [hunk 5]
```diff
@@ -9,31 +9,25 @@ OSS tools covered:
...
+> - developing multiple different ML models
```
Comment (m09) [inline README.md]:
```suggestion
> - developing multiple Machine Learning models
```

File: README.md [hunk 6]
```diff
@@ -9,31 +9,25 @@ OSS tools covered:
...
+> - for a particular task of source code summarization (or function name suggestion).
```
Comment (m09) [inline README.md:22]:
```suggestion
> - summarizing source code, suggesting function names and analyzing projects and developers most relevant topics
```

File: README.md [hunk 7]
```diff
@@ -9,31 +9,25 @@ OSS tools covered:
...
 >
-> - What information is available in your code
-> - How to extract this information
-> - What can you do with this knowledge: what are the tasks solvable by MLonCode
-> - Which models can be used to solve them
+> At the end of the workshop participants will build 2 working models on a real dataset, producing > near state-of-the-art results. Practical skill of extracting information from source code as well > as modeling different aspects of it are going to be acquired.
```
Comment (m09) [inline README.md]:
```suggestion
> At the end of the workshop participants will build 2 working models on a real dataset, producing near state-of-the-art results. Practical skill of extracting information from source code as well as modelling different aspects of it are going to be acquired.
```

**Review by m09 — COMMENTED** [review]
Minor things, LGTM otherwise

File: README.md (hunk 1 above)
Comment (bzz) [reply README.md]: 🤦‍♂ tnx!

File: README.md (hunk 6 above)
Comment (bzz) [reply README.md:22]: So far this text was 1:1 copy of the materials published on the website.
I'm not sure what's best to do in here

Comment (m09) [reply README.md:22]: It's probably fine to improve it here but I'm also fine keeping it as is.

**Review by m09 — COMMENTED** [review]
We should probably modify the Dockerfile too to keep the README synced with the rest of the repo (for the paths that we mount for example), and maybe some other files? Not sure about that.

**Comment by bzz** [comment]
> We should probably modify the Dockerfile too to keep the README synced with the rest of the repo (for the paths that we mount for example), and maybe some other files? Not sure about that.

Tnx, done in 3efc1c3. The root dir name is not changed yet.

**Pull request merged**